src/utils/api.js

```diff
@@ -14,4 +14,10 @@ export const getAllTopics = () => {
     return newsApi.get("/topics").then((res) => {
         return res.data
     })
+}
+
+export const getArticlesByTopic = (topic) => {
+    return newsApi.get(`/articles?topic=${topic}`).then((res) => {
+        return res.data
+    })
 }
```

Comment:
We can make our api request functions (I'm referring just to getArticlesByTopic here, to be clear) more DRY by using the axios params object in our getArticles get request options, I'd recommend Googling this and looking at the docs to try and implement this now as it will make things easier later on when we add more queries to our get articles endpoint!

Reply:
Hello, thanks for your advices.
I have added the param object inside getArticles and deleted getArticlesByTopic, and it works. So can I merge
 directly or resend the pull request?